start-preview.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+docker build -t kafka-site-preview .
+
+docker run -it --rm --name mypreview -p 8080:80 -v "$PWD":/usr/local/apache2/htdocs/ kafka-site-preview
```

Comment:
I needed to add :z to the volume mount for this to work with podman (as opposed to docker) on Fedora)

```suggestion
docker run -it --rm --name mypreview -p 8080:80 -v "$PWD":/usr/local/apache2/htdocs/:z kafka-site-preview
```